### ISSUE TITLE
Sigma value increases when training with no KL term.

### ISSUE THREAD

**Issue by N2606** [Leensman/VarDropPytorch#6]
Hi , 
In my intuition, sigma should be smaller with training go through. However, I found the sigma keep increasing so the loss term would increase after some epoch. (below figure.)
Is there some bugs in the code or it is just behave like that.
![image](https://user-images.githubusercontent.com/60968644/76823260-a567c300-684e-11ea-87a0-6d14db130e9a.png)
![image](https://user-images.githubusercontent.com/60968644/76823277-b3b5df00-684e-11ea-83de-568614049983.png)

**Comment by martinferianc** [comment]
Hey, maybe too late, but I have observed a similar problem and it is because of weight decay. In the original implementation, there is no weight decay (which of course makes sense) and here I think the author forgot to not include it for the ARD specific implementations.

**Comment by HolyBayes** [comment]
Thank you for your comment! Weight decay is removed
